### PR TITLE
fix(app-publishing): added metadata section required to publish to serverless repository

### PR DESCRIPTION
**Pull request by boltdynamics**
### Description
- Added metadata section to the sam template as its required to publish to the serverless repository, docs [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html#:~:text=Step%201%3A%20Add%20a%20Metadata%20section%20to%20the%20AWS%20SAM%20template).

Failed github action [run](https://github.com/Versent/stax-orchestrator/actions/runs/4643410709/jobs/8218873495).

### Contribution Type

- [ ] Issue (Feature)
- [x] Issue (Bug Fix)
- [ ] Security Update

### Testing
- Tested by publishing to serverless application repository. See github action [run](https://github.com/Versent/stax-orchestrator/actions/runs/4643940690/jobs/8218888248).

### CODE REVIEW

File: template.yml [hunk 1]
```diff
@@ -85,6 +85,22 @@ Parameters:
...
+      Stax Workloads Orchestrator -
+      Interact with Stax to CUD (create-update-delete)
+      workloads in your AWS Account(s).
+    Author: Versent
+    SpdxLicenseId: Apache-2.0
+    LicenseUrl: LICENSE
+    ReadmeUrl: README.md
+    Labels: ['AWS', 'Stax', 'Serverless', 'Lambda', 'StepFunction']
+    HomePageUrl: https://github.com/Versent/stax-orchestrator.git
+    SemanticVersion: 0.1.6
```
Comment (richardkeit) [inline template.yml:101]:
```suggestion
```
Let's remove this and allow it to be set by the build argument (CLI)

**Review by richardkeit — CHANGES_REQUESTED** [review]
Looks good, great to have it working
Small change on the app definition

File: template.yml (hunk 1 above)
Comment (boltdynamics) [reply template.yml]: Removed 👍

**Review by richardkeit — APPROVED** [review]
🚀

**Pull request merged**